### ISSUE TITLE
[BUG] syntax error near unexpected token `then'

### ISSUE THREAD

**Issue by mhaamann** [creyD/prettier_action#74]
I'm getting the following error when running with a standard prettier configuration
```
/home/runner/work/_actions/creyD/prettier_action/v4.1/entrypoint.sh: line 123: syntax error near unexpected token `then'
Error: Process completed with exit code 2.
```
Looks like the line was changed in commit dbdc55734ebeb55f608bde04f819938ecd8d906b any chance you can spot the error. It looks fine for me

Using the following .yml file:
```
name: Continuous Integration

on:
  pull_request:
    branches: [master]

jobs:
  prettier:
    runs-on: ubuntu-latest

    steps:
      - name: Checkout
        uses: actions/checkout@v2
        with:
          # Make sure the actual branch is checked out when running on pull requests
          ref: ${{ github.head_ref }}
          # This is important to fetch the changes to the previous commit
          fetch-depth: 0

      - name: Prettify code
        uses: creyD/prettier_action@v4.1
        with:
          prettier_version: '2.3.2'
          # This part is also where you can pass other options, for example:
          prettier_options: --write --config .prettierrc **/*.{js,jsx,less,ts,tsx}
          only_changed: True
```

**Comment by trevcarr95** [comment]
am also seeing the same issue when trying to upgrade to 4.1

**Comment by bananabrann** [comment]
> am also seeing the same issue when trying to upgrade to 4.1

Same here

**Comment by creyD** [comment]
@insyri this appears to be the line which we changed in the last update. Could you please take a look at it?

**Comment by insyri** [comment]
Noticed (5 mins ago, forgot to comment), I'm looking into this right now, my suspects are a syntactical error,

**Comment by creyD** [comment]
Closed in v4.1.1 thanks to @insyri!
